aesara/graph/basic.py

```diff
@@ -318,38 +318,38 @@ class Variable(Node):
     - `Constant`: a subclass which adds a default and un-replaceable
       :literal:`value`, and requires that owner is None.
 
-    - `TensorVariable` subclass of `Variable` that represents a `numpy.ndarray`
+    - `TensorVariable` subclass of `Variable` that represents a ``numpy.ndarray``
        object.
 
     - `TensorSharedVariable`: a shared version of `TensorVariable`.
 
     - `SparseVariable`: a subclass of `Variable` that represents
-      a `scipy.sparse.{csc,csr}_matrix` object.
+      a ``scipy.sparse.{csc,csr}_matrix`` object.
 
     - `GpuArrayVariable`: a subclass of `Variable` that represents our object on
-      the GPU that is a subset of `numpy.ndarray`.
+      the GPU that is a subset of ``numpy.ndarray``.
 
     - `RandomVariable`.
 
     A `Variable` which is the output of a symbolic computation will have an owner
     not equal to None.
 
-    Using the `Variables`' owner field and the `Apply` nodes' inputs fields,
+    Using a `Variable`\s' owner field and an `Apply` node's inputs fields,
     one can navigate a graph from an output all the way to the inputs. The
-    opposite direction is possible with a `FunctionGraph` and its
-    `FunctionGraph.clients` `dict`, which maps `Variable`s to a list of their
+    opposite direction is possible with a ``FunctionGraph`` and its
+    ``FunctionGraph.clients`` ``dict``, which maps `Variable`\s to a list of their
     clients.
 
     Parameters
     ----------
-    type : a Type instance
+    type: a Type instance
```

Comment:
You may have run into a [Sphinx bug](https://github.com/readthedocs/sphinx_rtd_theme/issues/766) here.
This led to confusion in PyMC3 a while back - see [my comment back then](https://github.com/pymc-devs/aesara/pull/256#issuecomment-753302196). For PyMC3 it's not that easy, because we use a custom doc them, but the Aesara doc theme is a standard one, so I would expect it to work with exact numpydoc style if the versions are correct.

The [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) is very clear about the spaces:
> The colon must be preceded by a space, or omitted if the type is absent.


Further below you _added_ the spaces...

Reply:
> You may have run into a [Sphinx bug](https://github.com/readthedocs/sphinx_rtd_theme/issues/766) here.

No, that bug was not involved.